core/templates/pages/exploration-player-page/layout-directives/learner-view-info.component.html

```diff
@@ -4,11 +4,11 @@
     <div class="lesson-info-section">
       <h1 class="oppia-exploration-h1">
         <span class="e2e-test-exploration-header oppia-exploration-header"
-              itemprop="description">
+              itemprop="description" title="{{ explorationTitle }}">
```

Comment:
```suggestion
              itemprop="description"
              title="{{ explorationTitle }}">
```

Reply:
Noted! and Updated PTAL! will upload at a screencast for the mobile view
Thanks!